Misc/NEWS.d/next/Library/2022-12-23-21-02-43.gh-issue-100474.gppA4U.rst

```diff
@@ -0,0 +1,2 @@
+Check that index page is actually a regular file before trying to serve it.
+This avoids issues around directories named "index.html".
```

Comment:
```suggestion
:mod:`http.server` now checks that an index page is actually a regular file before trying
to serve it.  This avoids issues with directories named `index.html`.
```

Reply:
Changing this because the news entry had no context about where the bug was.

Reply:
A sensible change.

Reply:
Thanks for the markup fix — been writing too much markdown!  🙁